staging/src/k8s.io/kubectl/pkg/describe/describe_test.go

```diff
@@ -5369,7 +5371,7 @@ Spec:
   PodSelector:     foo in (bar1,bar2),foo2 notin (bar1,bar2),id1=app1,id2=app2
   Allowing ingress traffic:
     To Port: 80/TCP
-    To Port: 82/TCP
+    To Port Range: 82-88/TCP
```

Comment:
if you modify current tests we loose coverage for the case someone specify only 2 ports, I think you just need to add new tests for port range

Reply:
Thanks for taking a look @aojea. Added a new test instead.